library/.eslintrc

```diff
@@ -28,18 +28,33 @@
   },
 
   "rules": {
-    "quotes": ["warn", "single", { "avoidEscape": true, "allowTemplateLiterals": true }],
     // http://eslint.org/docs/rules/
+
+    // "brace-style": ["warn", "1tbs", { "allowSingleLine": true }], // not sure what the correct config should be
```

Comment:
I think it indeed should be `1tbs`

Reply:
```js
// This is invalid (lib/node-utils.js:18)
try { ... }
catch (e) { ... }

// and should become:

try { ... } catch (e) { ... }

// or:

try {
  ...
} catch (e) { ... }


// This is also invalid (library/webpack-plugins/template-plugin.js:110)

x.then(y => { y.forEach(() =>
  ...
}) })

// and should become:

.then(x => {
  x.forEach(() =>
    ...
  })
})

Reply:
Hmm, that's tricky, I would be ok writing:

```
try x()
catch (e) y()
```
Which should be ok according to this rule, but is invalid javascript.

The second example seems to be a fair improvement.

Reply:
We'll enable the rule as discussed